rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/persistence/RedisBackedBakeStore.groovy

```diff
@@ -453,29 +456,17 @@ class RedisBackedBakeStore implements BakeStore {
 
   @Override
   public String deleteBakeByKey(String bakeKey) {
-    def jedis = jedisPool.getResource()
     def keyList = [bakeKey, "allBakes"]
-
-    jedis.withCloseable {
-      Set<String> incompleteBakesKeys = jedis.keys(allIncompleteBakesKeyPattern)
-
-      keyList += incompleteBakesKeys
-    }
+    keyList += this.scanIncompleteBakesKeys()
```

Comment:
minor stylistic nit - you don't need the explicit `this` qualifier to call the scanIncompleteBakesKeys method, can just write this as
```groovy
keyList += scanIncompleteBakesKeys()
```

Reply:
Thanks! I'm not sure I fully registered that this is a Groovy file 🙈